ports/portmidi/portfile.cmake

```
@@ -1,13 +1,15 @@
-vcpkg_fail_port_install(ON_TARGET "UWP")
+vcpkg_fail_port_install(ON_TARGET "linux" "osx" ON_ARCH "arm" ON_TARGET "uwp")
```

```suggestion
vcpkg_fail_port_install(ON_TARGET "linux" "osx" "uwp" ON_ARCH "arm")
```

I think this is nicer, imo.